packages/prisma-client/migrations-cli/umzug.ts

```diff
@@ -1,13 +1,39 @@
+/* eslint-disable no-console */
+
 import { Umzug, FileLocker, memoryStorage } from "umzug";
 import fs from "node:fs";
 import path from "node:path";
+import child_process from "node:child_process";
 import { prisma } from "../src";
 
-const executeFile = async (filepath: string, timeout: number = 1000 * 60) => {
+const runPrismaCLI = (args: string[]) => {
```

Comment:
did you check if prisma has an api we can import and do normal function calls instead of executing a script in a shell?